crates/re_arrow_store/src/store_stats.rs

```diff
@@ -2,13 +2,12 @@ use nohash_hasher::IntMap;
 use re_log_types::{ComponentName, DataCellColumn};
 
 use crate::{
-    store::IndexedBucketInner, DataStore, DataStoreConfig, IndexedBucket, IndexedTable,
-    PersistentIndexedTable,
+    store::IndexedBucketInner, DataStore, IndexedBucket, IndexedTable, PersistentIndexedTable,
 };
 
 // ---
 
-#[derive(Default, Debug, Clone)]
+#[derive(Default, Debug, Clone, PartialEq, Eq, PartialOrd)]
```

Comment:
does it make sense to order the stats?

Reply:
used in tests to check some basic invariants